src/Uno.UWP/ApplicationModel/Resources/ResourceLoader.cs

```diff
@@ -48,6 +48,23 @@ public ResourceLoader()
 
 		public string GetString(string resource)
 		{
+			
+			// "/[file]/[name]" format support
+			if(resource.ElementAt(0) == '/')
+			{
+				int iInd;
+				string sFile, sId;
+				sId = resource.Substring(1);
+				iInd = sId.IndexOf("/");
+				if(iInd > 0)
+				{
+					sFile = sId.Substring(0, iInd);
```

Comment:
Also, prefer inline declarations with the assignation, e.g.: 
```
					var resourceFile = sId.Substring(0, iInd);
```

Reply:
Why 'var', not 'string'?

Reply:
That's in the guidelines of the `.editorconfig` file. Prefer `var` when possible.